src/bitcoin-cli.cpp

```diff
@@ -34,7 +35,7 @@ std::string HelpMessageCli()
     strUsage += HelpMessageOpt("-datadir=<dir>", _("Specify data directory"));
     AppendParamsHelpMessages(strUsage);
     strUsage += HelpMessageOpt("-rpcconnect=<ip>", strprintf(_("Send commands to node running on <ip> (default: %s)"), DEFAULT_RPCCONNECT));
-    strUsage += HelpMessageOpt("-rpcport=<port>", strprintf(_("Connect to JSON-RPC on <port> (default: %u or testnet: %u)"), BaseParams(CBaseChainParams::MAIN).RPCPort(), BaseParams(CBaseChainParams::TESTNET).RPCPort()));
+    strUsage += HelpMessageOpt("-rpcport=<port>", strprintf(_("Connect to JSON-RPC on <port> (default: %u)"), defaultBaseParams->RPCPort()));
```

Comment:
And I don't like special cases for testnet3 (or any other chain), which I usually find with rgrep "test" or rgrep CBaseChainParams::TESTNET.
Maybe we can loop through all the default ports for each supported chain as in 
https://github.com/jtimon/bitcoin/commit/28d961ff939bfc0887c6c8b8b288733864559abd ?
I mean, that would require something like https://github.com/jtimon/bitcoin/commit/042fdb0f58313fbade482cc53e39aed0421437d8 first.